docs/releases/1.10.txt

```diff
@@ -544,6 +544,26 @@ This prevents confusion about an assignment resulting in an implicit save.
   :class:`~django.contrib.gis.geos.MultiPolygon` is deprecated in favor of the
   :attr:`~django.contrib.gis.geos.GEOSGeometry.unary_union` property.
 
+``CommaSeparatedIntegerField`` model field
+------------------------------------------
+
+``CommaSeparatedIntegerField`` is deprecated in favor of
+:class:`~django.db.models.CharField` with the
+:func:`~django.core.validators.validate_comma_separated_integer_list`
+validator::
+
+    from django.core.validators import validate_comma_separated_integer_list
+    from django.db import models
+
+    class MyModel(models.Model):
+        numbers = models.CharField(..., validators=[validate_comma_separated_integer_list])
+
+If you're using Oracle, ``CharField`` uses a different database field type
+(``NVARCHAR2``) from ``CommaSeparatedIntegerField`` (``VARCHAR2``). Therefore
+if your field stores more than 4000 bytes, use ``TextField`` (``NCLOB``)
+instead. If you have any queries that group by the field, you'll have to change
+them in that case.
```

Comment:
I think some more details are needed. I'd say, starting from the second line, instead of "Therefore":

Depending on your database settings, this may imply a different encoding, and thus, a different length (in bytes) for the same contents. In case this change makes some of your stored values cross the 4000 byte limit, use ``TextField`` (``NCLOB``) instead. If, in that case, you also have any queries that group by the field (e.g. annotating the model with an aggregation or using ``distinct()``) you will need to change them (to defer the field).